Makefile

```diff
@@ -37,6 +37,8 @@ check_license:
 	@./scripts/check_license.sh
 
 test:
+	@echo ">> installing consul binary used in tests"
+	@$(GO) get -u github.com/hashicorp/consul
```

Comment:
Ah, ok, I think I figured out a bit.

I'm not entirely sure about how to handle transitive dependencies. It pulls in a few. 

I guess the goal should be to make the tests run with "govendor test +local," is that right? I guess I need a bit more experimentation and getting used to this.